x/stake/client/rest/utils.go

```diff
@@ -49,6 +49,42 @@ func queryTxs(node rpcclient.Client, cliCtx context.CLIContext, cdc *codec.Codec
 	return tx.FormatTxResults(cdc, res.Txs)
 }
 
+func queryRedelegations(cliCtx context.CLIContext, cdc *codec.Codec, endpoint string) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		vars := mux.Vars(r)
+		bech32delegator := vars["delegatorAddr"]
+		bech32srcValidator := vars["srcValidatorAddr"]
+		bech32dstValidator := vars["dstValidatorAddr"]
+
+		delegatorAddr, err := sdk.AccAddressFromBech32(bech32delegator)
```

Comment:
```go
if err != nil {
			utils.WriteErrorResponse(w, http.StatusBadRequest, err.Error())
			return
		}
```

Reply:
@melekes  this comment would have been a great candidate for githubs new code suggestion feature!